utils/docker/Dockerfile.el.8

```diff
@@ -137,9 +137,9 @@ ARG JOBS=$DEPS_JOBS
 ARG DAOS_BUILD_TYPE=$DAOS_TARGET_TYPE
 ARG DAOS_BUILD=$DAOS_DEPS_BUILD
 
-# Build DAOS
+# Build DAOS with STACK_MMAP=1
 RUN [ "$DAOS_BUILD" != "yes" ] || {                                        \
-        scons --jobs $JOBS install PREFIX=/opt/daos COMPILER=$COMPILER     \
+        scons --jobs $JOBS install STACK_MMAP=1 PREFIX=/opt/daos COMPILER=$COMPILER     \
```

Comment:
why not just change the default?

Reply:
Until I complete https://daosio.atlassian.net/browse/DAOS-11547, we're concerned about the performance impact of `STACK_MMAP=1`